template/.github/workflows/android_deploy_production_to_playstore.yml

```diff
@@ -0,0 +1,50 @@
+name: Android - Deploy Production build to Play Store
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build_and_deploy_android:
+    name: Build & Deploy Android
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: ubuntu-latest
    timeout-minutes: 30
```

Reply:
Fixed in https://github.com/nimblehq/flutter-templates/pull/163/commits/f863a88ab487f8e40db08fa698a245029c73d346